src/pages/finance/paymentMethod/index.jsx

```diff
@@ -72,196 +61,138 @@ export const PaymentMethodDashboard = () => {
-                </Box>
-            </EmptyState>}
-
-
-            {lastPaymentMethods && lastPaymentMethods.length > 0 &&
-                <Box>
-                    <Typography variant={"h2"} fontSize={"1.5rem"}>Last used</Typography>
-                    <Box sx={{display: "flex", gap: "24px", mb: 4, mt: 2}}>
-                        {lastPaymentMethods.map(pm => {
-                            return (
-                                <Card sx={{border: "none", borderRadius: 4, flex: 1, maxWidth: "240px"}}
-                                      variant={"outlined"}>
-                                    <CardContent>
-                                        <Typography variant={"h3"} sx={{
-                                            alignItems: "center",
-                                            color: 'text.secondary',
-                                            display: "flex",
-                                            fontSize: 14,
-                                            gap: "8px",
-                                            mb: "16px"
-                                        }}>
-                                            {pm.type.toLowerCase().includes("card") ? <CreditCard/> : <LocalAtm/>}
-                                            {pm.type}
-                                        </Typography>
-
-                                        <Typography variant="caption" color={"text.secondary"}>
-                                            {pm.currency.symbol} - {pm.currency.name}
-                                        </Typography>
-                                        <Typography gutterBottom variant={"h3"} component="div">
-                                            {pm.name}
-                                        </Typography>
-
-                                        <Typography variant="caption" color={"text.secondary"}>
-                                            Used on - 01/01/1970
-                                        </Typography>
-                                    </CardContent>
-                                </Card>
-                            );
-                        })}
-                    </Box>
-                </Box>
+    }
+
+    const renderIconCards = (type) => {
+        switch (true) {
+            case type.toLowerCase().includes('card'):
+                return <CreditCard/>;
+            case type.toLowerCase().includes('cash'):
+                return <LocalAtm/>;
+            case type.toLowerCase().includes('bank'):
+                return <AccountBalance/>;
+            default:
+                return <Info/>;
+        }
+    }
+
+    const filterListOptions = [
+        {
+            label: "filter by type", type: "buttons", field: "type", options: typesOfPm && typesOfPm.map(t => {
+                return {label: t, value: t}
+            })
+        },
+        {
+            label: "filter by currency", type: "buttons", field: "currencyCode", options: [
```

Comment:
I would like to suggest to put a todo here to get that data from api when it is available